go-controller/pkg/cni/ovs.go

```diff
@@ -66,3 +68,30 @@ func ovsClear(table, record string, columns ...string) error {
 	_, err := ovsExec(args...)
 	return err
 }
+
+func ofctlExec(args ...string) (string, error) {
+	args = append([]string{"--timeout=30"}, args...)
+	output, err := runner.Command("ovs-ofctl", args...).CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("failed to run 'ovs-ofctl %s': %v\n  %q", strings.Join(args, " "), err, string(output))
+	}
+
+	outStr := string(output)
+	trimmed := strings.TrimSpace(outStr)
+	// If output is a single line, strip the trailing newline
+	if strings.Count(trimmed, "\n") == 0 {
+		outStr = trimmed
+	}
+
+	return outStr, nil
+}
+
+func waitForBrIntFlows(ip string) error {
+	return wait.PollImmediate(100*time.Millisecond, 20*time.Second, func() (bool, error) {
+		stdout, err := ofctlExec("dump-flows", "br-int", fmt.Sprintf("table=9,ip,nw_src=%s", ip))
+		if err != nil {
+			return false, nil
```

Comment:
Add a Infof level log indicating when the flow is not found.

Reply:
Disagree. Maybe one log before and one after but not one for each iteration.